app/elements/socobo-ranking/socobo-ranking.html

```diff
@@ -542,7 +546,7 @@
          */
         _reload: function() {
           // notify user
-          this.notify("Reloading...");
+          this.notify("Ranking - Reloading...");
```

Comment:
Maybe I would kill this toast and show only this one which display the successful end of the calculation. I would also remove the 'Ranking' phrase and change the text to something like 'Ranking  updated" or the like, since displaying only 'Calculation finished' the user does not know which calculation. I think this should be transparent to the user.